lib/node_modules/@stdlib/stats/base/meanpn/docs/types/index.d.ts

```diff
@@ -1,7 +1,7 @@
 /*
 * @license Apache-2.0
 *
-* Copyright (c) 2020 The Stdlib Authors.
+* Copyright (c) 2025 The Stdlib Authors.
```

Comment:
In general, unless you are adding new files, you don't need to update the copyright years of each file you touch.